src/widget/wkey.cpp

```diff
@@ -1,18 +1,25 @@
 #include "widget/wkey.h"
 
+#include <qboxlayout.h>
+#include <qevent.h>
```

Comment:
Why not
```suggestion
#include <QBoxlayout>
#include <QEvent>
```
like elsewhere?